sdcm/coredump.py

```diff
@@ -163,7 +179,6 @@ def _process_coredumps(
             except:  # pylint: disable=bare-except
                 pass
 
-    @retrying(n=10, sleep_time=20, allowed_exceptions=NETWORK_EXCEPTIONS, message="Retrying on getting pid of cores")
     def get_list_of_cores(self) -> Optional[List[CoreDumpInfo]]:
         return self._get_list_of_cores()
```

Comment:
If we removed `@retrying` then we don't need this wrapper at all? Probably, we need to rename `_get_list_of_cores()` to `get_list_of_cores()`?

Reply:
fixed